src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3532,6 +3532,10 @@ components:
           description: Date the snapshot was created
         profileId:
           $ref: '#/components/schemas/UniqueIdProperty'
+        source:
```

Comment:
I think this ends up growing the object quite a bit.  Do you think we could have an object that is just the region information?

Reply:
Do you mean making the `storage` object a top-level field or only including `storage` with the source?

Reply:
I think pulling the storage object to the top level so we could aggregate when we have more sources

Reply:
Thanks for clarifying! One downside though is you would have to access the storage information differently depending on whether you use the snapshot enumerate or detail endpoints.

Reply:
That's true...though we could potentially pull it up in detail too (but not necessarily). It's also a different object so from a client standpoint it'll be different. But that's a fair point.  I'm not sure if anyone depends on the structures matching